pkgs/servers/polaris/default.nix

```diff
@@ -10,13 +10,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "polaris";
-  version = "0.14.3";
+  version = "0.15.0";
 
   src = fetchFromGitHub {
     owner = "agersant";
     repo = "polaris";
-    rev = version;
-    hash = "sha256-2GHYIlEzRS7KXahdrxMjyIcPCNw8gXJw5/4ZpB/zT3Y=";
+    tag = version;
+    hash = "sha256-uwYNyco4IY6lF+QSVEOVVhZCJ4nRkj8gsgRA0UydLHU=";
 
     # The polaris version upstream in Cargo.lock is "0.0.0".
```

Comment:
It seems this isn't the case in the tagged release, so I think all this can go :) https://github.com/agersant/polaris/commit/11989aef00118addb8aa43295e785dd9352dcbd0